guides/release/components/template-lifecycle-dom-and-modifiers.md

```diff
@@ -485,58 +524,64 @@ export default modifier((element, [callback]) => {
 
 Now that we've created this modifier, we can use it in our `modal` component, and add some logic to invoke a passed-in action whenever the user clicks outside the modal.
 
-```handlebars {data-filename="app/components/modal.hbs"}
-<div class="modal" {{on-click-outside @clickedOutside}}>
+```gjs {data-filename="app/components/modal.gjs"}
+import onClickOutside from 'my-app/modifiers/on-click-outside';
```

Comment:
this is one modifier that I think I want to build in to the framework -- is a very common behavior

Reply:
That'd be useful.